features/support/object_model_pages/ivl/iap/ivl_iap_other_questions_page.rb

```diff
@@ -7,6 +7,14 @@ def self.is_pregnant_no_radiobtn
     'is_pregnant_no'
   end
 
+  def self.no_primary_caregiver_radiobtn
+    'input[id="is_primary_caregiver_no"]'
```

Comment:
we need to replace this with a `data-cuke` attribute

Reply:
Would this be an example of a data-cuke? `interaction-choice-control-value-is-primary-caregiver-no`

Reply:
`data-cuke="is_primary_caregiver_no"` on the same element as the id